docs/operator-manual/application.yaml

```diff
@@ -103,13 +103,31 @@ spec:
         beep: boop-${ARGOCD_APP_REVISION}
       # Toggle which enables/disables env variables substitution in commonAnnotations
       commonAnnotationsEnvsubst: true
+      labelWithoutSelector: false
```

Comment:
```suggestion
```

We don't support this field (yet). https://github.com/argoproj/argo-cd/blob/3c5ff60759abf9fbd6b8fdba7e05b0063a3604c8/pkg/apis/application/v1alpha1/types.go#L464-L493

Reply:
> We don't support this field (yet).
> 
> https://github.com/argoproj/argo-cd/blob/3c5ff60759abf9fbd6b8fdba7e05b0063a3604c8/pkg/apis/application/v1alpha1/types.go#L464-L493

Hello @crenshaw-dev thank you for you review.


I don't understand correctly, because I can to read LabelWithoutSelector on492 line

So Kustomize page it's wrong: https://argo-cd.readthedocs.io/en/stable/user-guide/kustomize/ ?